ui/de-webapp/src/main/resources/landing-page-template.html

```diff
@@ -2,14 +2,14 @@
 <html xmlns="http://www.w3.org/1999/xhtml">
 <head>
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
-    <title>iPlant Collaborative Discovery Environment</title>
+    <title>CyVerse Discovery Environment</title>
 
     <link href="$context_path$/landing-page-style.css" rel="stylesheet" type="text/css" />
 </head>
 
 <body>
 
-<img src="http://iplantc.org/sites/all/themes/iplant/images/iplant_logo.png" border="0" alt="Log in to iPlant" style="margin-left: 10px;" />
+<img src="$context_path$/cyverse_logo.png" border="0" height="107" width="500" alt="Log in to CyVerse" style="margin-left: 10px;" />
```

Comment:
:+1: -- this will probably remove those annoying mixed-content warnings on our homepage in many browsers!